packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+
+We prefer isvtest@okta.com to be a test user. If you can't use this email to create a test user, you can provide a specific account in a different domain.
+
+??? Perhaps remove/merge with the above ???
+
+Depending on the use case of our app, the permissions and roles for your test account are varied. For example:
+
+* For an SSO integration, you can provide a regular, non-admin user test account for the OIN team to test Single Sign-On with your app.
+* For a lifecycle management integration, you can provide a test account with HR admin privileges to onboard, change roles, or offboard employees on your app.
+* For an API service integration, you can provide a test account with privileges to trigger an API process on your app.
+
+> **Note:** Ensure that proper scopes are defined for OAuth 2.0 authentication.
+
+In general, the test account provides the OIN team with the ability to verify that your integration flow works as expected for your use case. Provide the necessary permissions required for your app. If you have further questions, contact the OIN team.
+
+## Customer configuration document guidelines
+
+A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
+
+You need to provide a configuration guide as part of the OIN submission process. Your guide is provided to administrators through the Okta Admin Console. Okta checks your document for general adherence to the configuration guide instructions. After your integration is published to the OIN, you can make the link public or customer‐accessible.
+
+>**Note:** A configuration guide is required for SCIM and OIDC integrations. It's optional for SAML integrations, as Okta supplies its own documentation with the apps.
+
+You can create the guide in whatever format works best for you: a webpage, a Google or Word doc, or a PDF are all acceptable.
+
+### Configuration guide examples
+
+OIDC:
+
+* [Cerby](https://docs.google.com/document/d/e/2PACX-1vRiswyNyRPVYfEMEwPsbMO8Qn11BjAO-FfUsWBBit_IYe88tzQCHTmMPtmF8uPV044HmXQR13adj3LO/pub)
+* [Upwave](https://help.upwave.io/en/articles/4129778-okta-configuration-guide)
+
+SAML:
+
+* [GitHub Enterprise](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Github_Enterprise.html)
+* [Runscope](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Runscope.html)
+* [Salesforce](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html)
+* [Zoom.us](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Zoom.us.html)
+
+SCIM:
+
+* [Atlassian Cloud](https://confluence.atlassian.com/cloud/configure-user-provisioning-with-okta-957492956.html)
+* [Contentful](https://www.contentful.com/help/okta-user-provisioning-integration-with-scim/)
+* [Zscaler](https://help.zscaler.com/zia/saml-scim-configuration-guide-okta)
+
+### Configuration guide content
```

Comment:
Should we put the examples below the guidelines to encourage people to actually look through these?

Reply:
Sure, I can do that. The doc examples were always presented first in the past (so I thought it was intentional).